src/util/loadExternals.ts

```diff
@@ -29,14 +27,16 @@ class CachedExternal {
   }
 }
 
-/* eslint-disable-next-line @typescript-eslint/no-explicit-any */
-function wrappedEval(content: string): any {
-  return eval(content);
+function wrappedEval(content: string): unknown {
+  // in case of a self-invoking wrapper, make sure self is defined
+  // as our context object.
+  return eval(`var self=this;\n${content}`);
```

Comment:
This is the most important change. Without this, some externals would still add their UMD global variable to the `window`, which is bad news for us.